src/lean_spec/subspecs/containers/state/state.py

```diff
@@ -86,12 +90,24 @@ def generate_genesis(cls, genesis_time: Uint64, num_validators: Uint64) -> "Stat
         )
 
         # Build the genesis block header for the state.
+        empty_attestations = Attestations(data=[])
+        empty_checkpoint = Checkpoint(root=Bytes32.zero(), slot=Slot(0))
+        proposer_attestation = ProposerAttestationData(
+            target=empty_checkpoint,
+            source=empty_checkpoint,
+        )
+
+        genesis_body = BlockBody(
+            attestations=empty_attestations,
```

Comment:
Super small nit but I find inlining easier to understand since `Attestations(data=[])` is self explanatory. But your call

```suggestion
            attestations=Attestations(data=[]),
```